scripts/globals/items/sprinters_shoes.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------------
+-- ID: 15754
+-- Item: Sprinter's Shoes
+-- Item Effect: Quickening for 60 minutes
+-----------------------------------------
+
+require("scripts/globals/settings");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    return 0;
+end;
+
+
+-----------------------------------------
+-- OnItemUse
+-----------------------------------------
+
+function onItemUse(target)
+    target:delStatusEffect(EFFECT_QUICKENING);
```

Comment:
no need to delStatusEffect, overwriting is baked into addStatusEffect and controlled by https://github.com/DarkstarProject/darkstar/blob/master/sql/status_effects.sql#L31

Reply:
https://github.com/DarkstarProject/darkstar/blob/master/src/map/status_effect.h#L34
```cpp
enum EFFECTOVERWRITE
{
    EFFECTOVERWRITE_EQUAL_HIGHER = 0, // only overwrite if equal or higher
    EFFECTOVERWRITE_HIGHER = 1, // only overwrite if higher
    EFFECTOVERWRITE_NEVER = 2, // never overwrite
    EFFECTOVERWRITE_ALWAYS = 3, // always overwrite no matter
    EFFECTOVERWRITE_IGNORE = 4 // ignore dupes
};
```

Reply:
Ahhh, yes deleting that line still has the effect be overwritten. Nice!

Reply:
But, how do I deal with the conflict with things that have flee? 
http://ffxiclopedia.wikia.com/wiki/Powder_Boots
```
Notes: Does not overwrite or stack with effects of Sprinter's Shoes.
```

Reply:
well those are 2 different effects. you just have one effect script check for the other (the effect, not the item) and behave accordingly. Or if the effect straight up won't apply that's even easier: we have an sql column for that.